easybuild/easyconfigs/p/PyOpenCL/PyOpenCL-2023.1.4-foss-2023a.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'PythonBundle'
+
+name = 'PyOpenCL'
+version = '2023.1.4'
+
+homepage = "https://mathema.tician.de/software/pyopencl/"
+description = """PyOpenCL lets you access GPUs and other massively parallel compute devices from Python."""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+dependencies = [
+    ('Python', '3.11.3'),
+    ('SciPy-bundle', '2023.07'),
+    ('pocl', '4.0'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('appdirs', '1.4.4', {
+        'checksums': ['7d5d0167b2b1ba821647616af46a749d1c653740dd0d2415100fe26e27afdf41'],
+    }),
```

Comment:
`appdirs` is provided by `Python-bundle-PyPI/2023.06-GCCcore-12.3.0`, which is a dependency of `SciPy-bundle`, so can be removed here

```suggestion
```